vertica_python/vertica/messages/frontend_messages/query.py

```diff
@@ -11,11 +11,8 @@ def __init__(self, query_string):
         self.query_string = query_string
 
     def to_bytes(self):
-        s = self.query_string
-        if isinstance(s, str):
-                s = str(s, 'utf-8')
-        encoded = s.encode('utf-8')
+        encoded = self.query_string.encode('utf-8')
```

Comment:
This essentially reverts 6a044c8718ed9ef34458735438fd7b354e286bb6. I'll add a dedicated test to make sure we don't drop support for unicode queries. I think however that the existing test for unicode named parameters already covers this, but better to explicit.